staging/src/k8s.io/mount-utils/go.mod

```diff
@@ -12,4 +12,7 @@ require (
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
 )
 
-replace k8s.io/mount-utils => ../mount-utils
+replace (
+	github.com/kr/pty => github.com/kr/pty v1.1.5
```

Comment:
> 1. these files should not be checked in and are left behind by buggy unit tests:  #98947
> 
> * pkg/util/iptables/xtables-103516112.lock
> * pkg/util/iptables/xtables-103657263.lock
> * pkg/util/iptables/xtables-104036897.lock

I did a fetch-rebase, reorged the commits to place the manual commit first.
then reran  `./hack/update-vendor.sh`, and pushed the branch again.
Those files are gone now.

> 1. This is a pretty huge diff grimacing

The _manual_ part of the PR is small: https://github.com/kubernetes/kubernetes/pull/98946/commits/707b1b905ffdfa16b508c420c28596bd424f90a6

The rest is all `update-vendor.sh`.  

> 2. Some of the dependencies appear to be under licenses that are not approved by the CNCF

That's unfortunate, as it would scuttle this upgrade.

> Does this totally resolve the k8sdeps thing?

Yes, it blows it away.

Reply:
Ah the `iptables` files are gone because i didnt't run `make test` locally a second time.

Reply:
Marking this resolved.  The commit links are out of date now.  Thanks @BenTheElder !